testsuite/tests/typing-jkind-bounds/subsumption/fuel.ml

```diff
@@ -0,0 +1,119 @@
+(* TEST
+    flags = "-extension layouts_alpha";
+    expect;
+*)
+
+type 'a my_list = 'a list = [] | ( :: ) of 'a * 'a my_list
+
+(* At the time this test is written, "fuel" is used to deal with recursive types. Two
```

Comment:
I think I don't understand this comment. The "below types"... what types specifically? The two types `t`, one of which is accepted and one of which isn't? They don't look equal to me: the first has 5 `list`s while the second has 6.

I think mostly this file is evidence that the fuel mechanism is not working out so well and that maybe we need to rethink. Or maybe instead that we shouldn't care so much. Not sure.

Reply:
If we do decide to keep this around, it might be worth getting this file to be generated, because we want the same types to be used throughout. There exist other tests that are generated. I could try to do some sleuthing to find them, but Chris probably knows off the top of his head.

Reply:
The numbers are different because I messed up - they're both meant to have 5 `list`s. I'll fix.